test/system/170-run-userns.bats

```diff
@@ -52,3 +52,15 @@ function _require_crun() {
     run_podman 125 run --group-add keep-groups --group-add 457 $IMAGE id
     is "$output" ".*the '--group-add keep-groups' option is not allowed with any other --group-add options" "Check group leaked into container"
 }
+
+@test "podman userns=auto in config file" {
+    skip_if_rootless "the user might not have enough IDs"
+    skip_if_remote "userns=auto is set on the server"
+
+    cat > $PODMAN_TMPDIR/userns_auto.conf <<EOF
+[containers]
+userns="auto"
+EOF
+    export CONTAINERS_CONF=$PODMAN_TMPDIR/userns_auto.conf
```

Comment:
Apologies for not spotting earlier. Could we instead of exporting set it as follows:
CONTAINERS_CONF=... run_podman

Otherwise, I am afraid of all subsequent calls using this containers.conf.

Reply:
fixed.

I wrongly assumed each test would run in its own process